src/main/webapp/app/types/sidebar.ts

```diff
@@ -91,6 +94,34 @@ export interface SidebarCardElement {
      * Set for Exercises. Will be removed after refactoring
      */
     exercise?: Exercise;
+    // TODO Extra Exam Part
+    /**
+     * This is a string which may define an icon for the status of the exam.
+     * It has to be a valid FontAwesome icon name and will be displayed in the
+     * 'regular' style. Needed for future implementation
+     */
+    statusIcon?: IconProp;
+    /**
+     * Set for Exam, identifies the color of the status icon. Needed for future implementation
+     */
+    statusIconColor?: string;
+    /**
+     * Set for Exam, shows the start date and time
+     */
+    startDateWithTime?: dayjs.Dayjs;
+    /**
+     * Set for Exam, shows the working time
+     */
+    workingTime?: number;
+    /**
+     * Set for exam, shows the maximum attainable Points
+     */
+    attainablePoints?: number;
+    /**
+     * Set for exam, indetifies the current status of an exam exercise for exam sidebar
+     */
+    rightIcon?: IconProp;
+    // TODO Extra Conversation Part
```

Comment:
Is the TODO accurate? Could you specify it a bit more closely?

Reply:
This todo is not written by me, but I think I can remove it for now since nothing is specified here.